src/css/flags/ads.less

```diff
@@ -33,10 +33,14 @@
         }
     }
 
-    .jw-logo,
-    .jw-captions, video::-webkit-media-text-track-container {
```

Comment:
Just add `.jw-captions-enabled` to `.jw-captions` instead of making a new block:
```css
.jw-logo, .jw-captions.jw-captions-enabled, video::-webkit-media-text-track-container {
```
That's all the `&` operator in less does, but making a new block adds it as another rule.